x/evidence/client/cli/query_test.go

```diff
@@ -60,7 +60,9 @@ func TestGetQueryCmd(t *testing.T) {
 			true,
 		},
 		"all evidence (default pagination)": {
-			[]string{},
+			[]string{
+				fmt.Sprintf("--%s=text", flags.FlagOutput),
+			},
```

Comment:
I think depending on the order of tests run, its possible for this test to return as json if the json test runs before it, so we just specify for this to return as text